### ISSUE TITLE
Dashboard status buton does not match the real status

### ISSUE THREAD

**Issue by jvrrr** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#651]
Hi,

Sorry for my laziness ;-) , but I described the issue, with pictures in the Domoticz forum and they said to me it is een plugin issue (sorry)

https://www.domoticz.com/forum/viewtopic.php?f=6&t=36375

Can you help me out?

**Comment by hughb8on** [comment]
Hi, is your plugin code up to date? I ask because a fix was applied 10 days ago which inverted blind on/off behaviour and I guess it may be relevant to your problem.
See issue #638.

**Comment by jvrrr** [comment]
> Hi, is your plugin code up to date? I ask because a fix was applied 10 days ago which inverted blind on/off behaviour and I guess it may be relevant to your problem.
> See issue #638.

Yes I did. I posted this: https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues/613
And the solution was what you posted.

I updated the plugin -> restarted  zigbee2mqtt and Domoticz -> removed the switch en reinstalled it in Domoticz.
The switches are not inverted anymore.

But the status of the Dashboard still not match with the switch status.

Thanks!!!
